src/composables/ViewRoutingSlip/useLinkedRoutingSlipsPaymentInfo.ts

```diff
@@ -0,0 +1,27 @@
+import { Payment } from '@/models/Payment'
+import { PaymentMethods } from '@/util/constants'
+import { RoutingSlip } from '@/models/RoutingSlip'
+import { createNamespacedHelpers } from 'vuex-composition-helpers'
+
+const routingSlipModule = createNamespacedHelpers('routingSlip') // specific module name
+const { useState } = routingSlipModule
+
+// Composable function to inject Props, options and values to PaymentDetails component
+export default function useLinkedRoutingSlipsPaymentInfo () {
```

Comment:
This is WIP component. Would be adding code for displaying children payment information in here